src/Protos/Grpc/cluster.proto

```diff
@@ -0,0 +1,138 @@
+﻿syntax = "proto3";
+package event_store.cluster;
+
+import "shared.proto";
```

Comment:
Some of this is snake_case and some of it is pascalCase

Reply:
I believe snake case is preferred for interop @thefringeninja ?